nix/marlowe-cardano/deploy/operables.nix

```diff
@@ -360,6 +361,7 @@ in
 
   marlowe-tx = mkOperableWithProbes {
     package = marlowe-tx;
+    runtimeInputs = [ z3 ];
```

Comment:
@jhbertra, is this the only derivation where the new runtime dependency of `marlowe-tx` on `z3` will need to be added?

Reply:
I think this needs to be added to `compose.nix` as well. You could add this to `run-local-service`:

```nix
export PATH="$PATH:${lib.makeBinPath [ z3 ]}"
```

Reply:
Fixed in 34f1d7e5d89e68996254f165386a3264372c3252.